src/main/java/duke/Panorama.java

```diff
@@ -0,0 +1,109 @@
+package duke;
+
+import java.util.Scanner;
+import java.util.List;
+
+import java.util.Locale;
+
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeParseException;
+
+// Task folder
+import duke.MyTask.Task;
+
+// Exceptions folder
+import duke.MyException.EmptyDescriptionException;
+import duke.MyException.UnknownCommandException;
+
+public class Panorama {
+    static TaskList taskList;
+    static Ui ui;
+
+    /**
+     * @return true if BYE command was executed, false otherwise
+     */
+    static boolean handleCommand(String input) throws EmptyDescriptionException, UnknownCommandException {
+        String[] tokens = input.split(" ");
+        Command command = Command.fromString(tokens[0]);
+
+        switch (command) {
+            case MARK:
+                taskList.markTask(tokens[1]);
+                break;
+            case UNMARK:
+                taskList.unmarkTask(tokens[1]);
+                break;
+            case BYE:
+                ui.exit_greeting();
+                return true;
+            case LIST:
+                taskList.listEntries();
+                break;
+            case TODO:
+                taskList.addTodoTask(input);
+                break;
+            case DEADLINE:
+                taskList.addDeadlineTask(input);
+                break;
+            case EVENT:
+                taskList.addEventTask(input);
+                break;
+            case DELETE:
+                taskList.deleteTask(tokens[1]);
+                break;
+            case HELP:
+                ui.display_help();
+                break;
+            default:
+                throw new UnknownCommandException();
+        }
```

Comment:
I believe based on the coding standards there shouldn't be indentations for case 😄